package.json

```diff
@@ -94,6 +102,30 @@
     "react-app-rewired": "^2.0.0",
     "react-scripts": "^2.0.3",
     "rimraf": "^2.6.2",
-    "source-map-explorer": "^1.6.0"
+    "source-map-explorer": "^1.6.0",
+    "ts-jest": "^23.10.3",
+    "ts-loader": "^5.2.1",
+    "typescript": "^3.1.1"
+  },
+  "jest": {
```

Comment:
If jest supports loading from a separate config file in this particular scenario, I'd prefer that. Not a huge fan of putting all the config in one file.

Reply:
It doesn't. I tried.